app/scripts/views/mixins/resume-token-mixin.js

```diff
@@ -20,13 +20,15 @@ define(function (require, exports, module) {
     getResumeToken: function () {
       // there might not be any relier if the resume token is being fetched
       // for an account unlock request caused by changing the password.
+      var flowInfo = this.flow && this.flow.pickResumeTokenInfo();
```

Comment:
Oh, this fills in a blank for me. It's the first time I've noticed this function, before I thought the resume token was created  out of thin air and magic.